File: decompiled/sources/com/hjq/http/exception/ResponseException.java
package com.hjq.http.exception;

import okhttp3.Response;

public final class ResponseException extends HttpException {
    private final Response mResponse;

    public ResponseException(String str, Response response) {
        super(str);
        this.mResponse = response;
    }

    public ResponseException(String str, Throwable th, Response response) {
        super(str, th);
        this.mResponse = response;
    }

    public Response getResponse() {
        return this.mResponse;
    }
}

File: decompiled/sources/com/hjq/http/exception/ResponseException.kt
package com.hjq.http.exception

import okhttp3.Response

class ResponseException : HttpException {
    private val mResponse: Response

    constructor(str: String, response: Response) : super(str) {
        mResponse = response
    }

    constructor(str: String, th: Throwable, response: Response) : super(str, th) {
        mResponse = response
    }

    fun getResponse(): Response {
        return mResponse
    }
}
